error_reporting/google/cloud/error_reporting/__init__.py

```diff
@@ -14,5 +14,32 @@
 
 """Client library for Stackdriver Error Reporting"""
 
+from google.cloud.error_reporting.client import Client, HTTPContext
 
-from google.cloud.error_reporting.client import Client
+
+def build_flask_context(request):
+    """Builds an HTTP context object from a Flask (Werkzeug) request object.
+
+     This helper method extracts the relevant HTTP context from a Flask request
+     object into an object ready to be sent to Error Reporting.
+
+     Example::
+
+        >>> @app.errorhandler(Exception)
+        >>> def handle_error(e):
+        >>>     client = error_reporting.Client()
+        >>>     client.report_exception(
+        >>>         http_context=build_flask_context(request))
+        >>>     # rest of error response code here
+
+    :type request: :class:`werkzeug.wrappers.request`
+    :param request: The Flask request object to convert.
+
+    :rtype: :class:`google.cloud.error_reporting.client.HTTPContext`
+    :returns: An HTTPContext object ready to be sent to the Stackdriver Error
+              Reporting API.
+    """
+    return HTTPContext(url=request.url, method=request.method,
+                       user_agent=str(request.user_agent),
```

Comment:
Actually I don't believe it is. I can't find it in the flask docs but [this SO question](http://stackoverflow.com/a/33706555/89702) talks about it a little bit.